flow/include/flow/serialize.h

```diff
@@ -887,7 +887,12 @@ class MakeSerializeSource : public ISerializeSource {
 public:
 	using value_type = V;
 	void serializePacketWriter(PacketWriter& w) const override {
-		ObjectWriter writer([&](size_t size) { return w.writeBytes(size); }, AssumeVersion(w.protocolVersion()));
+		ObjectWriter writer(
+		    +[](size_t size, void* pPacketWriter) {
+			    return (*reinterpret_cast<PacketWriter*>(pPacketWriter)).writeBytes(size);
```

Comment:
```suggestion
			    return static_cast<PacketWriter*>(pPacketWriter)->writeBytes(size);
```

cf. https://stackoverflow.com/a/310489 e.g.

Reply:
This is a good point to know. I am certain I read this page before but I did not remember.